packages/gatsby/cli.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+
+require(`./dist/bin/gatsby.js`)
```

Comment:
Alternatively - delete https://github.com/gatsbyjs/gatsby/blob/e0ad4212d68ea41e3eba91de6591d251bd309df5/packages/gatsby/src/bin/gatsby.js#L1-L3

and import `gatsby-cli` here, but I personally would keep it in case some script use this one

Reply:
No, you're right. We should keep it in case some tooling tries to import that directly